curriculum/challenges/english/20-upcoming-python/learn-string-manipulation-by-building-a-cipher/65524d2a1a253b8bb5197ae2.md

```diff
@@ -0,0 +1,36 @@
+---
+id: 65524d2a1a253b8bb5197ae2
+title: Step 21
+challengeType: 20
+dashedName: step-21
+---
+
+# --description--
+
+As you can see, uppercase letters are still an issue. In fact, `find` returns `-1` for them, and for the space character, too. You are going to take care of the space later on.
+
+For now, instead of looping over `text`, try to iterate over `text.lower()`.
```

Comment:
```suggestion
`find` is again returning `-1` for uppercase letters, and for the space character, too. You are going to take care of the space later on.

For now, instead of looping over `text`, change the `for` loop to iterate over `text.lower()`.
```